source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
I think this ought to go into a separate pull request because it is logically independent. But I'm doubtful that that is correct -- the bottom right block of the Stokes system is empty, so why do you need this entry?